cloud/passportauth/integration/UserSeedData.ts

```diff
@@ -22,7 +23,8 @@ const userSeedData: User[] = [
     password: '123',
     phone: '2657258272',
     position: 'Senior Truck Driver',
-    username: 'trever'
+    username: 'trever',
+    roles: []
```

Comment:
We know that we going to support 'User' and 'Admin" role so best to set them here.